virtme_ng_init/src/main.rs

```diff
@@ -686,6 +686,15 @@ fn setup_network() -> Vec<Option<thread::JoinHandle<()>>> {
 
     let cmdline = std::fs::read_to_string("/proc/cmdline").unwrap();
     if cmdline.contains("virtme.dhcp") {
+        // Make sure all GIDs are allowed to create raw ICMP sockets (this allows to run ping as
+        // regular user).
+        utils::run_cmd(
+            "bash",
+            &[
+                "-c",
+                &"sysctl -w net.ipv4.ping_group_range=\"0 2147483647\"",
+            ],
+        );
```

Comment:
Would it not be better to write text in `/proc/sys/net/ipv4/ping_group_range` instead of calling bash + sysctl?

It's a detail, if you prefer the current version, fine for me.

Reply:
Good idea, I also prefer to write to procfs instead of using sysctl, I'll change that, thanks!